app/components/widgets/forms/rich-text-editor.js

```diff
@@ -63,12 +63,11 @@ export default Component.extend({
       const updateValue = () => {
         debounce(this, () => {
           const value = this.editor.getValue();
-          const description = String(value);
-          let x = description.length;
-          while (description.substr(0, x).endsWith('<br>')) {
-            x = x - 4;
+          let description = String(value);
+          while (description.endsWith('<br>')) {
+            description = description.replace(/<br>$/, '');
```

Comment:
That's not what I meant. There should be no loop

Reply:
removed that in the final commit already.